### PR TITLE
Improve alter edge

### LINKED ISSUE #349: Implement ALTER EDGE according to ALTER TAG
subtask of #240

### PR DESCRIPTION
**Pull request by steppenwolfyuetong**
Implement ALTER EDGE according to ALTER TAG
support alter edge sql

This PR will close #349

### CODE REVIEW

**Comment by dutor** [comment]
Jenkins, please run tests.

File: src/meta/client/MetaClient.cpp [hunk 1]
```diff
@@ -719,7 +720,8 @@ MetaClient::createEdgeSchema(GraphSpaceID spaceId, std::string name, nebula::cpp
 }
 
 folly::Future<StatusOr<bool>>
-MetaClient::alterEdge(GraphSpaceID spaceId, std::string name,
+MetaClient::alterEdgeSchema(GraphSpaceID spaceId,
+                      std::string name,
```
Comment (laura-ding) [inline src/meta/client/MetaClient.cpp]: alignment

Comment (steppenwolfyuetong) [reply src/meta/client/MetaClient.cpp]: Good catch， 👍
ps : Last night I found a flaw in the original code, I submitted it together.

**Comment by steppenwolfyuetong** [comment]
Fix error using typeId function， please review again.

**Comment by dutor** [comment]
Jenkins, please run tests

**Pull request merged**